uiplib/scheduler.py

```diff
@@ -19,11 +19,25 @@ def __init__(self, offline, pics_folder, timout, website, count):
         self.timeout = timout
         self.directory = pics_folder
         self.count = count
+
         if not offline:
-            fetch = Thread(target=self.initFetch)
-            # all child threads need to be daemons to die upon main thread exit
-            fetch.setDaemon(True)
-            fetch.start()
+            try:
+                #Init the threads
+                unsplashThread = Online_Fetch('unsplash',self.website[0],
```

Comment:
can we make this as a loop? like i said, what if i have 3 websites in my `settings.json` ? we need to see the url and see what kind of url it is reddit or unsplash and then put them in the function